src/golibmc.go

```diff
@@ -24,6 +24,7 @@ const (
 	PollTimeout    = C.CFG_POLL_TIMEOUT
 	ConnectTimeout = C.CFG_CONNECT_TIMEOUT
 	RetryTimeout   = C.CFG_RETRY_TIMEOUT
+	MaxRetries     = C.CFG_MAX_RETRIES
```

Comment:
This is not a compatible change. Don't forget to bump the major version. At least @jumpeiMano will be affected.

Reply:
Okay, I will. You mean the `ConfigNotWaitForRetryTimeout` interface right? I think they will get error in compile time, so it's ok.